Makefile

```diff
@@ -218,7 +218,6 @@ DOC_FILES_14 += $(GENERATED_MD_FILES_14)
 $(GENERATED_MD_FILES_14): | generated-md-1.4
 $(DOC_MARKER_14): $(DOC_FILES_14) | github-wiki
 	$(PYTHON) $(DMLC_DIR)/validate_md_links.py $(DOC_SRC_DIR_14)/toc.json $(DOC_SRC_DIR_14) generated-md-1.4
-	$(PYTHON) $(DMLC_DIR)/md_to_github.py $(DOC_SRC_DIR_14)/toc.json github-wiki.tar.gz $(DOC_SRC_DIR_14) generated-md-1.4
```

Comment:
If we don't build the wiki as part of DMLC, then what does build it? Or would we have to do it manually? Would this break Andreas' work on automatically updating the wiki upon releases?

Reply:
This PR should be merged after [this one](https://github.com/intel-innersource/applications.simulators.simics.simics-base/pull/1418), which causes the Simics build system to generate the wiki.